provider/testdata/src/metadata.rs

```diff
@@ -2,49 +2,23 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-use cargo_metadata::{self, camino::Utf8PathBuf, MetadataCommand};
 use displaydoc::Display;
 use icu_locid::LanguageIdentifier;
 use serde::Deserialize;
 
-#[derive(Display, Debug)]
-#[non_exhaustive]
-pub enum Error {
-    #[displaydoc("Cargo Error: {0}")]
-    Cargo(cargo_metadata::Error),
-    #[displaydoc("Serde Error: {0}")]
-    SerdeJson(serde_json::Error),
-    #[displaydoc("Package not found")]
-    PackageNotFound,
-    #[displaydoc("package.metadata.icu4x_testdata not found")]
-    MetadataNotFound,
-}
-
-impl std::error::Error for Error {}
-
-impl From<cargo_metadata::Error> for Error {
-    fn from(e: cargo_metadata::Error) -> Self {
-        Error::Cargo(e)
-    }
-}
-
-impl From<serde_json::Error> for Error {
-    fn from(e: serde_json::Error) -> Self {
-        Error::SerdeJson(e)
-    }
-}
+static SOURCES_TOML: &str = include_str!("../sources.toml");
```

Comment:
thought: in the long run we should probably figure out a way to do this that gets clients the ICU tag and the CLDR version without needing to pull in the toml crate.

either via a build script or via const trickery (`include!()` the file, write a simple const parser that finds the version lines)